link_shortener/templates/all_links.html

```diff
@@ -9,6 +9,9 @@
       </a>
     </div>
     <br><br>
+    {% for owner in data["owners"] %}
+      {{ owner }}
+    {% endfor %}
```

Comment:
Tohle prosím zakomentuj. Nechme to připravený pro FE ale nezobrazoval bych to

Reply:
Klidně k tomu připiš komentář pro Duca

Reply:
Jasne.
926612062375581f8151eb259c2a56dea48ceb63

Reply:
Zakomentuj to prosím HTMLkově, přes <!-- -->. Bude to číst frontendista 🙂

Reply:
Done.
6be4d42975e43930b96e2e00b10c19b21efacb59

Reply:
Super, jen to máš nějak divně indentnutý, a je tam zapomenutá řádka s #}, ale to už snad přežiju

Reply:
cleanupnute
e38e783d8d97c25c8f1c0f1ae1bab12532e29979

Reply:
3e0cf5aa84e9138503811547f7b985aaef212a84